biscuit/src/Auth/Biscuit/Datalog/AST.hs

```diff
@@ -744,7 +770,7 @@ renderRuleScope =
   let renderScopeElem = \case
         OnlyAuthority -> "authority"
         Previous      -> "previous"
-        BlockId bs    -> "ed25519/hex:" <> Hex.encodeBase16 (pkBytes bs)
+        BlockId bs    -> "ed25519/" <> Hex.encodeBase16 (pkBytes bs)
```

Comment:
The spec has changed in the meantime?

Reply:
yes https://github.com/biscuit-auth/biscuit/pull/117